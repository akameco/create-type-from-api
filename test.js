// @flow
const fs = require('fs')
const m = require('.')

test('snapshot', () => {
  const data = fs.readFileSync('./__fixtures__/data.json', 'utf8')
  expect(m(JSON.parse(data))).toMatchSnapshot()
})
