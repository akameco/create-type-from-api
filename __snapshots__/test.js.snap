// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
"{
  data: {
    ranking: [
      {
        title: String
        id: Number
        imageUrls: {
          medium: String
          large: String
        }
      }
    ]
  }
}"
`;
